instance/models/openedx_appserver.py

```diff
@@ -295,40 +294,317 @@ def get_playbooks(self):
+            "COMMON_ENABLE_NEWRELIC_APP": True if settings.NEWRELIC_LICENSE_KEY else False,
+            "NEWRELIC_LICENSE_KEY": settings.NEWRELIC_LICENSE_KEY if settings.NEWRELIC_LICENSE_KEY else "",
+
+            # Discovery
+            "SANDBOX_ENABLE_DISCOVERY": False, # set to true to enable discovery
+            "DISCOVERY_ELASTICSEARCH_URL": 'http://localhost:9200',
+            "DISCOVERY_URL_ROOT": 'https://{}'.format(self.instance.discovery_domain),
+            "DISCOVERY_HOSTNAME": '~{}'.format(self.instance.discovery_domain_nginx_regex),
+            "DISCOVERY_NGINX_PORT": 80,
+            "DISCOVERY_SSL_NGINX_PORT": 443,
+
+            # Ecommerce
+            "SANDBOX_ENABLE_ECOMMERCE": False,  # set to true to enable ecommerce
+            "ECOMMERCE_NGINX_PORT": 80,
+            "ECOMMERCE_SSL_NGINX_PORT": 443,
+            "ECOMMERCE_HOSTNAME": '~{}'.format(self.instance.ecommerce_domain_nginx_regex),
+            "ECOMMERCE_ECOMMERCE_URL_ROOT": 'https://{}'.format(self.instance.ecommerce_domain),
+            "EDXAPP_ECOMMERCE_PUBLIC_URL_ROOT": 'https://{}'.format(self.instance.ecommerce_domain),
+            "EDXAPP_ECOMMERCE_API_URL": 'https://{}/api/v2'.format(self.instance.ecommerce_domain),
+            "ECOMMERCE_SUPPORT_URL": 'https://{}'.format(self.instance.domain),
+            "ECOMMERCE_LMS_URL_ROOT": 'https://{}'.format(self.instance.domain),
+            "ECOMMERCE_COURSE_CATALOG_URL": 'https://{}'.format(self.instance.discovery_domain),
+
+            "ECOMMERCE_PLATFORM_NAME": '"{{ EDXAPP_PLATFORM_NAME }}"',
+            # Set your own ECOMMERCE_PAYMENT_PROCESSOR_CONFIG, or update the variables for the appropriate processor:
+            # * CyberSource
+            "ECOMMERCE_CYBERSOURCE_PROFILE_ID": 'SET-ME-PLEASE',
+            "ECOMMERCE_CYBERSOURCE_MERCHANT_ID": 'SET-ME-PLEASE',
+            "ECOMMERCE_CYBERSOURCE_ACCESS_KEY": 'SET-ME-PLEASE',
+            "ECOMMERCE_CYBERSOURCE_SECRET_KEY": 'SET-ME-PLEASE',
+            "ECOMMERCE_CYBERSOURCE_TRANSACTION_KEY": 'SET-ME-PLEASE',
+            "ECOMMERCE_CYBERSOURCE_PAYMENT_PAGE_URL": 'https://set-me-please',
+            "ECOMMERCE_CYBERSOURCE_RECEIPT_PAGE_URL": '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/',
+            "ECOMMERCE_CYBERSOURCE_CANCEL_PAGE_URL": '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/',
+            "ECOMMERCE_CYBERSOURCE_SOAP_API_URL": 'https://set-me-please',
+            # * PayPal
+            "ECOMMERCE_PAYPAL_MODE": 'SET-ME-PLEASE',
+            "ECOMMERCE_PAYPAL_CLIENT_ID": 'SET-ME-PLEASE',
+            "ECOMMERCE_PAYPAL_CLIENT_SECRET": 'SET-ME-PLEASE',
+            "ECOMMERCE_PAYPAL_RECEIPT_URL": '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/',
+            "ECOMMERCE_PAYPAL_CANCEL_URL": '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/',
+            "ECOMMERCE_PAYPAL_ERROR_URL": '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/error/',
+
+            # Comprehensive theming
```

Comment:
Can you please remove the commented out code?  It's unclear why it is there, and we have git to recover it if we need it.

Reply:
Good point, I just kept everything from the original file. These should be removed now.